### PR TITLE
Update FanGraphs docs, and improve typing in FG code

### PR DESCRIPTION
**Pull request by TheCleric**

### CODE REVIEW

File: docs/fangraphs.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+fangraphs.fg_team_fielding_data(2019)
+```
+
+## Team Pitching Stats
+
+```python
+from pybaseball.datasources import fangraphs
+
+fangraphs.fg_team_pitching_data(2019)
+```
```
Comment (schorrm) [inline docs/fangraphs.md:61]: Combine to one code block

Comment (schorrm) [reply docs/fangraphs.md:61]: And can we use the top level usage here as the example?

File: docs/fangraphs.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+# Team Batting Stats
+fangraphs.fg_team_batting_data(2019)
+
+# Team Fielding Stats
+fangraphs.fg_team_fielding_data(2019)
+
+# Team Pitching Stats
+fangraphs.fg_team_pitching_data(2019)
+```
```
Comment (schorrm) [inline docs/fangraphs.md:42]: I prefer `pybaseball.fg_batting_stats` as more continuous with the established API

Comment (TheCleric) [reply docs/fangraphs.md:42]: Do you want me to add extra scope to this PR to fix that then? Or add a separate PR for that.

Comment (schorrm) [reply docs/fangraphs.md:42]: I thought this was the case prior honestly... I guess you could just throw it in here

Comment (schorrm) [reply docs/fangraphs.md:42]: Hmm... this is already the case actually, checking out the `__init__.py`. I'm gonna modify this so I can merge now

**Comment by schorrm** [comment]
Went thru the CLI due to the conflicts.

**Pull request closed without merging**